frontend/packages/network-attachment-definition-plugin/src/components/network-attachment-definitions/NetworkAttachmentDefinitionsForm.tsx

```diff
@@ -27,11 +27,17 @@ import { networkTypeParams, networkTypes } from '../../constants';
 import NetworkTypeOptions from './NetworkTypeOptions';
 
 const buildConfig = (name, networkType, typeParamsData): NetworkAttachmentDefinitionConfig => {
-  const config: NetworkAttachmentDefinitionConfig = {
-    name,
-    type: networkType,
-    cniVersion: '0.3.1',
-  };
+  const config: NetworkAttachmentDefinitionConfig =
+    networkType === 'cnv-bridge'
```

Comment:
If we need this `if` here and on line 49, maybe it would make sense to just combine them? The current state of the code is a little hard to follow. And if we extend it with additional types, it may bring a lot of `if`s.

Just a suggestion, won't fight for that

Reply:
:+1: fixed